InAppUtils/InAppUtils.m

```diff
@@ -190,7 +190,52 @@ - (void)paymentQueueRestoreCompletedTransactionsFinished:(SKPaymentQueue *)queue
     }
 }
 
+RCT_EXPORT_METHOD(refreshReceipt:(RCTResponseSenderBlock)callback)
+{
+    [self refreshReceipt:callback testExpired:NO testRevoked:NO];
+}
+
+RCT_EXPORT_METHOD(testRefreshExpiredReceipt:(RCTResponseSenderBlock)callback)
```

Comment:
instead of exposing three methods can you expose one with options object for expired and revoked?

something like :
```js
refreshReceipt({ expired: true }, callback);
```

Reply:
Honestly I can't figure out how to do this without also requiring an empty dictionary to be passed in when neither are required, which is the most common case. I am not sure if this is the best interface in that case. I would be possible to do if we also had a js layer that abstracted the objc layer but as it stands, I don't know how to do this in a clean way.